dolphinscheduler-bom/pom.xml

```diff
@@ -391,7 +391,7 @@
                 <groupId>mysql</groupId>
                 <artifactId>mysql-connector-java</artifactId>
                 <version>${mysql-connector.version}</version>
-                <scope>test</scope>
+                <scope>compile</scope>
```

Comment:
Need to switch back to `test` because of incompatible license.

Reply:
ok, i'll switch it back when DRAFT Label removed